HDDS-7357 Update release page for 1.3.0.

captainzmc opened the pull request:
Add release page for Ozone 1.3.0 release to the master branch.
Just as what we did in 1.2.0 release: https://github.com/apache/ozone-site/pull/17

adoroszlai commented on content/faq.md:
```diff
@@ -31,7 +31,7 @@ Please take a look at the getting started document. For the impatient, we have d
 There are instructions on how to start Ozone on conventional machines or VMs if you would like to go that way.
 
-You can [download](/downloads/) it and [try it out.](/docs/1.2.0/start/runningviadocker.html)
+You can [download](/downloads/) it and [try it out.](/docs/1.3.0/start/runningviadocker.html)
```
I think we can fix this for good by:

```suggestion
You can [download](/downloads/) it and [try it out.](/docs/current/start/runningviadocker.html)
```

adoroszlai commented on content/release/1.3.0.md:
```diff
@@ -0,0 +1,42 @@
+---
+title: Release 1.3.0 available
+date: 2021-11-30
```
```suggestion
date: 2022-11-30
```

adoroszlai commented on content/release/1.3.0.md:
```diff
@@ -0,0 +1,42 @@
+title: Release 1.3.0 available
+date: 2021-11-30
+linked: true
+aliases:
+    - /release/1.3.0
```
Why is this alias needed?  Seems to be the same as the real name.

errose28 replied:
Yeah this was used [previously](https://github.com/apache/ozone-site/commit/89cdb07b59cc6610f5d0402b846c750b6c8b28a2) to redirect 1.2.0 to 1.2.1. We shouldn't need it here for 1.3.0.

adoroszlai commented on content/release/1.3.0.md:
```diff
@@ -0,0 +1,42 @@
+It represents a point of API stability and quality that we consider production-ready. 
+
+
+Image credit: [Glacier National Park 8][image] by [Tony Hisgett][author], [CC BY 2.0][cc] / Text added to original
+
+[image]: https://www.flickr.com/photos/hisgett/8051784706/
```
Alt text and picture should refer to Grand Canyon National Park, not Glacier.

adoroszlai commented on content/release/1.3.0.md:
```diff
@@ -0,0 +1,42 @@
+Apache Ozone 1.3.0 is released with following features:
+
+- Erasure Coding - V1 support (Online reconstruction support)
+- Erasure Coding - offline recovery support (HDDS-6462)
```
EC offline recovery support is not complete, I don't think we should advertise it yet.

CC @sodonnel

adoroszlai commented on content/release/1.3.0.md:
```diff
@@ -0,0 +1,42 @@
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+![glacier-haiku](releases/1.3.0.jpg)
```
Not sure how useful the alt text "glacier-haiku" is, but it should refer to Grand Canyon.

adoroszlai reviewed (changes requested):
Thanks a lot @captainzmc for working on this (and the entire release process).

errose28 commented on content/release/1.3.0.md:
```diff
@@ -0,0 +1,42 @@
+- Erasure Coding - offline recovery support (HDDS-6462)
+- Container balancer
+- Limit number of rocksdb instances to 1 per disk on high-density datanodes (HDDS-3630)
+- Multi-tenancy Support for Ozone (S3 multi-volume support)
```
```suggestion
- Ozone S3 Multi-tenancy Support
```

errose28 commented on content/release/1.3.0.md:
```diff
@@ -0,0 +1,42 @@
+- Limit number of rocksdb instances to 1 per disk on high-density datanodes (HDDS-3630)
+- Multi-tenancy Support for Ozone (S3 multi-volume support)
+- S3 Grpc improvements for metadata path
+- Support filesystem optimization for existing installations not already using it
```
```suggestion
- Filesystem Optimized and Object Store bucket layout types
```

errose28 reviewed (commented):
Thanks for the website updates and release management work @captainzmc

captainzmc commented:
Thanks to @adoroszlai and @errose28  for your review. We plan to prepare RC2, and we can solve these problems in RC2. I will update this PR then.

captainzmc commented:
RC2 vote of 1.3.0 passed, we have updated maven and svn, now we can update the release page. 
Hi @adoroszlai @errose28, I have fixed review issues, Could you help please help take another look?

adoroszlai commented on content/release/1.3.0.md:
```diff
@@ -19,24 +17,23 @@ aliases:
-[author]: https://www.flickr.com/people/37804979@N00
+[image]: https://www.flickr.com/photos/sarhounds/50193622148/
+[author]: https://www.flickr.com/people/sarhounds
 [cc]: https://creativecommons.org/licenses/by/2.0/deed.en
```
The license for this image does not allow reuse, we need to [find another](https://www.flickr.com/search/?text=%22grand%20canyon%22&license=2%2C3%2C4%2C5%2C6%2C9) with more permissive license.

How about [this one](https://www.flickr.com/photos/geewhypics/50720710687)?

captainzmc replied:
Sure. Thanks @adoroszlai. I'll update this image.

captainzmc commented:
@adoroszlai Image has been updated.

adoroszlai reviewed (commented):
Thanks @captainzmc for updating the patch, LGTM.

@errose28 would you like to take another look?

errose28 reviewed (approved):
Thanks for the site updates @captainzmc

captainzmc commented:
Thanks @adoroszlai @errose28 for the review. Let's merge this.
